blockchain/state_manager/src/lib.rs

```diff
@@ -1163,10 +1176,10 @@ where
         loop {
             select! {
                 res = subscriber_poll => {
-                    return res;
+                    return res.map_err(|e| Error::Other(format!("failed joining on tokio task {e}")))?
```

Comment:
Is it possible to write a `From` implementation that does this error conversion automatically?

Reply:
Added